src/govuk/components/accordion/accordion.js

```diff
@@ -93,49 +105,55 @@ Accordion.prototype.initSectionHeaders = function () {
 
 // Set individual header attributes
 Accordion.prototype.initHeaderAttributes = function ($headerWrapper, index) {
-  var $module = this
   var $span = $headerWrapper.querySelector('.' + this.sectionButtonClass)
   var $heading = $headerWrapper.querySelector('.' + this.sectionHeadingClass)
   var $summary = $headerWrapper.querySelector('.' + this.sectionSummaryClass)
 
   // Copy existing span element to an actual button element, for improved accessibility.
   var $button = document.createElement('button')
-  $button.setAttribute('type', 'button')
```

Comment:
This is setting the button's`type`, not `role`.

I think there's _potential_ for this change to cause issues for accordions inside forms, as the default `type` for a button is `submit` and I can't see anything in the JS to prevent the default click behaviour on the button which would be to submit the form.

Reply:
Yes, agreed – this would be a breaking bug for anywhere accordions are used inside forms. My service current does this, although we’re in the process of moving away from using the accordion in this way.

Reply:
All makes sense, restored. 🙌🏼